crl/x509/crl.go

```diff
@@ -46,15 +45,16 @@ type RevokedCertificate struct {
 	// value of 0 represents a reasonCode extension containing enum value 0 (this
 	// SHOULD NOT happen, but can and does).
 	ReasonCode *int
-	// Extensions contains all raw extensions parsed from the CRL entry, or all
-	// extra extensions to add to the CRL entry. When creating a CRL, if
-	// Extensions contains a reasonCode extension, it will be ignored in favor of
-	// the ReasonCode field above.
-	Extensions []pkix.Extension
+	// When creating a CRL, ExtraExtensions should contain all extra extensions to
+	// add to the CRL entry. If ExtraExtensions contains a reasonCode extension,
+	// it will be ignored in favor of the ReasonCode field above. When parsing a
+	// CRL, ExtraExtensions contains all raw extensions parsed from the CRL entry,
+	// except for reasonCode which is represented by the ReasonCode field above.
+	ExtraExtensions []pkix.Extension
```

Comment:
This seems like a reversion; In one of the later commits in #6198 you [changed from ExtraExtensions to Extensions](https://github.com/letsencrypt/boulder/pull/6198/commits/ca72086c9dadb0cd600e20dd76ee4a30cad91253). Why go back to ExtraExtensions again?